guides/plugins/plugins/framework/message-queue/add-message-to-queue.md

```diff
@@ -0,0 +1,120 @@
+# Add message to queue
+
+## Overview
```

Comment:
I'm missing a general explanation, what a the message queue is. 

The message queue allows you to ... 

Maybe you can copy something from the symfony docs as some sort of introduction here